recipes/vterm-toggle

```diff
@@ -0,0 +1,6 @@
+(vterm-toggle :fetcher github
+	:repo "jixiuf/vterm-toggle"
+    :files ("*"
+        (:exclude "LICENSE" ".dir-locals.el" ".gitignore" ".clang-format" ".travis.yml"))
```

Comment:
This repository feels like it could use the [single file recipe](https://github.com/melpa/melpa#example-single-file-repository) unless I'm missing something.